examples/split-rendering-rtt/main.js

```diff
@@ -395,22 +396,38 @@
             scene.addChild(model1);
 
             // create textured quad with the texture rtt
-            // var cameraScreen = this.createCamera( opaque );
-            // this._root.addChild( cameraScreen );
-
-            // composer
-            var composer = new osgUtil.Composer();
-            var blendPass = new osgUtil.Composer.Filter.Custom(composeFragmentShader, {
-                Texture1: transparent,
-                Texture0: opaque
-            });
-
-            blendPass.getOrCreateStateSet().setTextureAttributeAndModes(1, transparent);
-            blendPass.getOrCreateStateSet().setTextureAttributeAndModes(0, opaque);
-            composer.addPass(blendPass);
-            composer.renderToScreen(this._canvas.width, this._canvas.height);
-
-            this._root.addChild(composer);
+            var blendRTT = true;
+            if (blendRTT) {
```

Comment:
Always true? no need for the if/else block

Reply:
was in the demo before, I think it's for debug or smth. 
didn't want change to change behavior

Reply:
@Kuranes I guess is better to remove it. If it does not make sense, better to get rid.